### DISCUSSION TITLE
关于m3u8播放的问题

### DISCUSSION THREAD

**Discussion by zhangde2333** [hooke007/mpv_PlayKit#419] [mpv 通用]
大佬们，我是新手，想问一下，我使用猫爪抓出来的m3u8链接，使用mpv播放，是可以一直播放的，但是每次播12秒（一个片段）会停顿加载一下，然后会继续播放一个12秒的片段。我应该怎么修改来实现一个一直播放，后台加载的效果，感谢

**Comment by zhangde2333** [comment]
![image](https://github.com/hooke007/MPV_lazy/assets/51786345/5bbdff02-be7a-4b81-a9fe-516ef59005ba)

**Comment by hooke007** [comment]
m3u8是播放列表，如果 https://mpv.io/manual/master/#options-prefetch-playlist 没用，那我就不知道了

**Reply by zhangde2333** [reply]
好像只能这样了，感谢感谢
